src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindOutputPresenter.java

```diff
@@ -302,8 +302,12 @@ public void execute()
                         for (String pattern : dialogState_.getExcludeFilePatterns())
                            excludeFilePatterns.push(pattern);
 
-                        server_.completeReplace(dialogState_.getQuery(),
-                                                dialogState_.isRegex(),
+                        String serverQuery = dialogState_.getQuery();
```

Comment:
Previously, we were padding with `\b` in the find operation but not in the complete replace operation, which is why the find and preview respected the whole word option, but complete replace did not. This fixes that issue. 

This bit could have been separated out into its own PR, since it was strictly a failure to pass the correct args to the RPC, but for purposes of testing I've included it together with the other changes.

Reply:
Nice catch!